spec/features/courses/show_spec.rb

```diff
@@ -26,34 +26,36 @@
     )
   end
 
+  let(:course_page) { PageObjects::Page::Organisations::Course.new }
+
   scenario 'viewing the show courses page' do
     visit "/organisations/A0/courses/#{course.attributes[:course_code]}"
 
-    expect(find('.govuk-caption-xl')).to have_content(
+    expect(course_page.caption).to have_content(
       course.attributes[:description]
     )
-    expect(find('.govuk-heading-xl')).to have_content(
+    expect(course_page.title).to have_content(
```

Comment:
How do you differentiate between the title element and the H1?

Reply:
`title` here can be renamed to `heading` perhaps, it's just a hook anyway. It can be anything we like, doesn't automatically mean `title` is the `<title>`